### PR TITLE
Pact Broker Chart

### PR DESCRIPTION
**Pull request by gsampaio-rh**

### CODE REVIEW

**Comment by eformat** [comment]
hi .. may wanna change route.yaml template to

spec:
  host: null

this will autogenerate hostname

File: pact-broker/templates/route.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{{- if .Values.route }}
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: {{ .Values.appName }}
+  labels:
+    app: {{ .Values.appName }}
+spec:
+  host: "teste.com"
```
Comment (eformat) [inline pact-broker/templates/route.yaml]: change to

  host: null

File: pact-broker/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  labels:
+    app: {{ .Values.appName }}
+spec:
+  ports:
+    - name: {{ .Values.service.port | default "8080" }}-tcp
+      port: {{ .Values.service.port | default "8080" }}
+      protocol: TCP
+      targetPort: {{ .Values.service.port | default "8080" }}
+  selector:
+    app: {{ .Values.appName }}
```
Comment (eformat) [inline pact-broker/templates/service.yaml]: this end up pointing to the database pod ? i had to change to:

  selector:
    deploymentconfig: {{ .Values.appName }}

File: pact-broker/templates/postgres-dc.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          persistentVolumeClaim:
+            claimName: postgresql-{{ .Values.appName }}-data
+  triggers:
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+          - postgresql
+        from:
+          kind: ImageStreamTag
+          name: postgresql:{{ .Values.postgresql.image.tag | default "9.5" }}
```
Comment (eformat) [inline pact-broker/templates/postgres-dc.yaml]: 9.6 ? in ocp4 9.5 does not exist

File: pact-broker/values.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  enabled: true
+
+postgresql:
+  accessmode: ReadWriteOnce
+  service:
+    port: 5432 # default
+    database: *name
+    type: ClusterIP
+  image:
+    tag: "9.5"
```
Comment (eformat) [inline pact-broker/values.yaml]: 9.6 ? in ocp4 9.5 does not exist

**Comment by gsampaio-rh** [comment]
@eformat Hey Mike, is it ok to get this error when host is set to null?
helm install pact-broker -f pact-broker/values.yaml pact-broker                                                                                           ✔  11154  18:53:53
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Route.spec): missing required field "host" in com.github.openshift.api.route.v1.RouteSpec

**Comment by eformat** [comment]
LGTM

**Pull request merged**